src/journal-remote/journal-remote.conf.in

```diff
@@ -18,3 +18,7 @@
 # ServerKeyFile={{CERTIFICATE_ROOT}}/private/journal-remote.pem
 # ServerCertificateFile={{CERTIFICATE_ROOT}}/certs/journal-remote.pem
 # TrustedCertificateFile={{CERTIFICATE_ROOT}}/ca/trusted.pem
+# MaxUse=
+# KeepFree=
+# MaxFileSize=
+# MaxFiles=100
```

Comment:
Nowhere the default `100` is set in the code. I think it is better to drop it (also drop the value mentioned in the man page) to keep the backward compatibility.

Reply:
I've removed it here.